---
layout: archive
title: "Publications"
permalink: /publications/
author_profile: true
---


<!-- {% include base_path %}
I will try adding pdf for all the papers asap :) Alternatively,  you can find almost all
pdfs from my Google Scholar. If you cannot find any, please feel free to
email me.

I am also working on a Dataset page - we
always support data sharing in our community.


Publications
===== -->
Lab members are **bold**.

† denotes equal contribution   * denotes co-corresponding authors.

used Xin Maizie Zhou (X. M. Zhou) for publications after 2022.

## Preprints/Under Review

- **Brain structure and activity predicting cognitive maturation in adolescence**  
  J. Zhu, C. M. Garin, X.-L. Qi, A. Machado, Z. Wang, S. B. Hamed, T. R. Stanford, E. Salinas, C. T. Whitlow, A. W. Anderson, **X. M. Zhou**, F. Calabro, B. Luna, C. Constantinidis  
  [_bioRxiv_ (2024)](https://www.biorxiv.org/content/10.1101/2024.08.23.608315v1)

- **stDyer enables spatial domain clustering with dynamic graph embedding**  
  K. Xu, Y. Xu, Z. Wang, **X. M. Zhou**, L. Zhang  
  [_bioRxiv_ (2024)](https://www.biorxiv.org/content/10.1101/2024.05.08.593252v2)

- **Leveraging cross-source heterogeneity to improve the performance of bulk gene expression deconvolution**  
  W. Shen*, C. Liu, **Y. Hu**, Y. Lei, H.-S. Wong, S. Wu*, **X. M. Zhou**  
  [_bioRxiv_ (2024)](https://www.biorxiv.org/content/10.1101/2024.04.07.588458v1)

- **CNVeil enables accurate and robust tumor subclone identification and copy number estimation from single-cell DNA sequencing data**  
  **W. Yuan**, **C. Luo**, **Y. Hu**, L. Zhang, Z.-H. Wen, **Y. H. Liu**, X. Mallory, **X. M. Zhou**  
  [_bioRxiv_ (2024)](https://www.biorxiv.org/content/10.1101/2024.02.21.581409v1)

- **MaskGraphene: Advancing joint embedding, clustering, and batch correction for spatial transcriptomics using graph-based self-supervised learning**  
  **Y. Hu**, Y. Li, **M. Xie**, M. Rao, **Y. H. Liu**, **X. M. Zhou**  
  [_bioRxiv_ (2024)](https://www.biorxiv.org/content/10.1101/2024.02.21.581387v1)

- **Large indel detection in region-based phased diploid assemblies from linked-reads**  
  **C. Luo**, B. A. Peters, **X. M. Zhou**  
  [_BMC Genomics_ (under revision) (2024)](https://www.biorxiv.org/content/10.1101/2024.02.21.581409v1)

## Peer-reviewed Publications

### 2024
- **VolcanoSV enables accurate and robust structural variant calling in diploid genomes from single-molecule long read sequencing**  
  **C. Luo†**, **Y. H. Liu†**, **X. M. Zhou**  
  _Nature Communications_ (2024) 15:6956. [DOI](https://doi.org/10.1038/s41467-024-51282-0)

- **Benchmarking clustering, alignment, and integration methods for spatial transcriptomics**  
  **Y. Hu**, **M. Xie**, Y. Li, M. Rao, W. Shen, **C. Luo**, **H. Qin**, **J. Baek**, **X. M. Zhou**  
  _Genome Biology_ (2024) 25:212. [DOI](https://doi.org/10.1186/s13059-024-03361-0)

- **SCCNAInfer: A robust and accurate tool to infer the absolute copy number on scDNA-seq data**  
  L. Zhang, **X. M. Zhou**, X. Mallory  
  _Bioinformatics_ (2024) 40(7), btae454. [DOI](https://doi.org/10.1093/bioinformatics/btae454)

- **Parallel signatures of cognitive maturation in primate antisaccade performance and prefrontal activity**  
  J. Zhu, **X. M. Zhou**, C. Constantinidis, E. Salinas, T. R. Stanford  
  _iScience_ (2024) 27(8):110488. [DOI](https://doi.org/10.1016/j.isci.2024.110488)

- **Tradeoffs in alignment and assembly-based methods for structural variant detection with long-read sequencing data**  
  **Y. H. Liu†**, **C. Luo†**, S. G. Golding, J. B. Ioffe, **X. M. Zhou**  
  _Nature Communications_ (2024) 15:2447. [DOI](https://doi.org/10.1038/s41467-024-46614-z)

### 2023
- **Laminar pattern of adolescent development changes in working memory neuronal activity**  
  J. Zhu, B. Hammond, **X. M. Zhou**, C. Constantinidis  
  _Journal of Neurophysiology_ (2023) 130(4):980-989.

- **Editorial: predicting high-risk individuals for common diseases using multi-omics and epidemiological data, volume II**  
  W. P. Veldsman, **X. M. Zhou**, Y. Zhang, B. Li, L. Zhang.
  _Front. Genet._ (2023) 14:1280648.

- **ADEPT: Autoencoder with differentially expressed genes and imputation for robust spatial transcriptomics clustering**  
  **Y. Hu†**, Y. Zhao†, C. T. Schunk, Y. Ma, T. Derr*, **X. M. Zhou***  
  _iScience_ (2023) 26(6):106792. [DOI](https://doi.org/10.1016/j.isci.2023.106792)

- **Haplotyping-assisted diploid assembly and variant detection with linked-reads**  
  **Y. Hu**, C. Yang, L. Zhang*, **X. Zhou***  
  _Methods in Molecular Biology_ (2023) 2590:161-182. doi: 10.1007/978-1-0716-2819-5_1. PMID: 36335499.

### 2022
- **Haplotype-phasing of long-read HiFi data to enhance structural variant detection through a Skip-Gram model**  
  **C. Luo**, **P. A. Datar**, **Y. H. Liu**, **X. Zhou**  
  _IEEE International Conference on Bioinformatics and Biomedicine (BIBM)_ (2022), Las Vegas, NV, USA, 2022, pp. 2326-2333, doi: 10.1109/BIBM55620.2022.9995293.

- **A comprehensive investigation of statistical and machine learning approaches for predicting complex human diseases on genomic variants**  
  C. Wang, J. Zhang, **X. Zhou**, L. Zhang. 
  _IEEE International Conference on Bioinformatics and Biomedicine (BIBM)_ (2022), Las Vegas, NV, USA, 2022, pp. 2326-2333, doi: 10.1109/BIBM55620.2022.9995293.

- **Automated filtering of genome-wide large deletions through an ensemble deep learning framework**  
  **Y. Hu**, **S. Mangal**, L. Zhang, **X. Zhou**.
  _IEEE International Conference on Bioinformatics and Biomedicine (BIBM)_ (2022), Las Vegas, NV, USA, 2022, pp. 2326-2333, doi: 10.1109/BIBM55620.2022.9995293.

- **Graphing cell relations in spatial transcriptomics**  
  **X. Zhou**  
  _Nature Computational Science (2022) 2, 354-355_  
  [DOI: 10.1038/s43588-022-00269-2](https://doi.org/10.1038/s43588-022-00269-2)