---
layout: archive
title: "Software"
permalink: /Software/
author_profile: true
---


This page contains the tools and software packages we have developed @ Maize Zhou Lab or previously @ Stanford.

Please visit our [Lab GitHub page](https://github.com/maiziezhoulab) for the latest releases.

- [**BenchmarkST**](https://github.com/maiziezhoulab/BenchmarkST): Benchmarking clustering, alignment, and integration methods for spatial transcriptomics.
- [**CNVeil**](https://github.com/maiziezhoulab/CNVeil): Accurate and robust tumor subclone identification and copy number estimation from single-cell DNA sequencing data.
- [**VolcanoSV**](https://github.com/maiziezhoulab/VolcanoSV): Accurate and robust structural variant calling in diploid genomes from single-molecule long-read sequencing.
- [**MaskGraphene**](https://github.com/maiziezhoulab/MaskGraphene): Advancing joint embedding, clustering, and batch correction for spatial transcriptomics using graph-based self-supervised learning.
- [**RegionIndel**](https://github.com/maiziezhoulab/RegionIndel): Large indel detection in region-based phased diploid assemblies from linked-reads.
- [**ADEPT**](https://github.com/maiziezhoulab/ADEPT): Autoencoder with differentially expressed genes and imputation for a robust spatial transcriptomics clustering.
- [**LRSV_combo**](https://github.com/maiziezhoulab/LRSV_combo): Long-read based SV calling tools analysis.
- [**embSV**](https://github.com/maiziezhoulab/embSV): Haplotype-phasing of long-read HiFi data to enhance structural variant detection through a Skip-Gram model.
- [**AquilaDeepFilter**](https://github.com/maiziezhoulab/AquilaDeepFilter): Automated filtering of genome-wide large deletions through an ensemble deep learning framework.
- [**Bfimpute**](https://github.com/maiziezhoulab/Bfimpute): A Bayesian factorization method to recover single-cell RNA sequencing data.
- [**RNN_workingmemoryaccuracy**](https://github.com/maiziezhoulab/RNN_workingmemoryaccuracy): Neural mechanisms of working memory accuracy revealed by recurrent neural networks.
- [**RNN_BrainMaturation**](https://github.com/maiziezhoulab/RNN_BrainMaturation): Emergence of prefrontal neuron maturation properties by training recurrent neural networks in cognitive tasks.
- [**Autism_genepheno**](https://github.com/maiziezhoulab/Autism_genepheno): Text mining of gene-phenotype associations reveals new phenotypic profiles of autism-associated genes.
- [**MARS**](https://github.com/maiziezhoulab/MARS): A package for multiple samples alignment-based structural variant calling and refinement.
- [**Aquila_stLFR**](https://github.com/maiziezhoulab/Aquila_stLFR): Diploid genome assembly-based structural variant calling package for stLFR linked-read.
- [**Aquila**](https://github.com/maiziezhoulab/Aquila): Diploid personal genome assembly and comprehensive variant detection based on linked-reads.
- [**HAPDeNovo**](https://github.com/maiziezhoulab/HAPDeNovo): A haplotype-based approach for filtering and phasing de novo mutations in linked-read sequencing data.